scripts/json_schema/gen_json_schema.py

```diff
@@ -154,9 +154,11 @@ def get_required(self, obj):
         file_name = file_name.replace("Schema", "")
         schema_list.append(target_schema["definitions"][cls.__name__])
         print(target_schema)
-    schema = {
-        "type": "object",
-        "oneOf": schema_list
-    }
-    with open((f"Flow.schema.json"), "w") as f:
-        f.write(json.dumps(schema, indent=4))
+        with open((f"{file_name}.schema.json"), "w") as f:
+            f.write(json.dumps(target_schema, indent=4))
+    # schema = {
```

Comment:
why this comment here?

Reply:
Commented to dump single EagerFlow's schema

Reply:
Reverted